### PR TITLE
Palak_Assignment_1

### PR DESCRIPTION
**Pull request by PalakSJun20**
image

### CODE REVIEW

File: src/palak/FormAssignment1.java [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+			System.out.println("Test Fail Title is not " + title);
+
+		// Fill all the form
+		driver.findElement(By.id("first name")).sendKeys("Palash");
+		driver.findElement(By.id("last name")).sendKeys("Soni");
+		driver.findElement(By.id("E-mail")).sendKeys("Palash444@gmail.com");
+		driver.findElement(By.id("Company Name")).sendKeys("Vodafone");
+		
+		//Selecting Gender Value
+		WebElement genderValue = driver.findElement(By.name("gender"));
```
Comment (KrishnaKTechnocredits) [inline src/palak/FormAssignment1.java:40]: using this name which value are you selecting, is not clear !!!

File: src/palak/FormAssignment1.java [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+		confirmDetailValue.click();
+		if (confirmDetailValue.isSelected()) {
+			System.out.println("Test Pass\n7.Confirm Details is selected");
+		} else
+			System.out.println("Test Fail\n7. Confirm details is not selected");
+
+		// Reset Functionality
+		WebElement reset = driver.findElement(By.id("resetBtn"));
+		reset.click();
+		if (driver.findElement(By.id("Company Name")).getAttribute("value").isEmpty())
```
Comment (KrishnaKTechnocredits) [inline src/palak/FormAssignment1.java:97]: Where is the validation for radio button and checkbox ?

File: src/palak/FormAssignment1.java (hunk 1 above)
Comment (PalakSJun20) [reply src/palak/FormAssignment1.java]: Changes Done
`if(driver.findElement(By.id("femaleG")).isSelected());
		 	driver.findElement(By.id("maleG")).click();`

File: src/palak/FormAssignment1.java (hunk 2 above)
Comment (PalakSJun20) [reply src/palak/FormAssignment1.java]: Added 
if (driver.findElement(By.id("Company Name")).getAttribute("value").isEmpty() && (!dreamCompanyValue.isSelected())
					&&(!learningLangValue.isSelected()))

**Pull request merged**